src/libs/protocols/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "protocols"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = []
+with-serde = [ "serde", "serde_json" ]
+
+[dependencies]
+ttrpc = { git = "https://github.com/haosanzi/ttrpc-rust", features = ["async"] }
```

Comment:
> Could you specify a version for this forked repo? Also, is there an issue that you could add as a comment here for tracking the removal of this fork?

Ok，sure.  If the Occlum supports the Unix Socket, it will be replaced with official crate.